src/arrayMethodShift.js

```diff
@@ -5,7 +5,17 @@
  */
 function applyCustomShift() {
   [].__proto__.shift2 = function() {
-    // write code here
+    if (this.length === 0) {
+      return undefined;
+    }
+    const element = this[0];
```

Comment:
какой именно элемент?
```suggestion
    const firstElement = this[0];
```

Reply:
> какой именно элемент?

ok